vcr/signature/ldutils.go

```diff
@@ -28,12 +28,27 @@ import (
 	"net/url"
 )
 
+// JSONLDContextsConfig contains config for json-ld document loader
+type JSONLDContextsConfig struct {
+	// RemoteAllowList A list with urls as string which are allowed to request
+	RemoteAllowList []string `koanf:"remoteallowlist"`
+	// LocalFileMapping contains a list of context urls mapped to a local file
+	LocalFileMapping map[string]string `koanf:"localmapping"`
+}
+
 // embeddedFSDocumentLoader tries to load documents from an embedded filesystem.
```

Comment:
```suggestion
// embeddedFSDocumentLoader loads documents from an embedded filesystem.
```

Reply:
We dont' know if the given file is on the embedded FS, so I prefer trying?